### PR TITLE
Add misc rewards fixes

### PR DESCRIPTION
**Pull request by piazzatron**
### Description
- Removes extraneous REWARDS_IDS remote config (also removed on optimizely)
- `isComplete` was broken for boolean challenges, where the stepCount never increments. 

### Dragons

None

### How Has This Been Tested?

Locally

### How will this change be monitored?

n/a

### CODE REVIEW

**Review by dylanjeffers — APPROVED** [review]
LGTM

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/aud-1992

File: src/common/services/remote-config/defaults.ts [hunk 1]
```diff
@@ -33,8 +33,6 @@ export const remoteConfigStringDefaults: {
   // Audius user id
   [StringKeys.TRENDING_PLAYLIST_OMITTED_USER_IDS]: '51',
   // Remove when Rewards UI update is live
```
Comment (Kyle-Shanks) [inline src/common/services/remote-config/defaults.ts:35]: Can remove this comment too to avoid confusion

**Review by Kyle-Shanks — APPROVED** [review]

File: src/common/services/remote-config/defaults.ts (hunk 1 above)
Comment (piazzatron) [reply src/common/services/remote-config/defaults.ts:35]: nice

**Pull request merged**